pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarMetadata.java

```diff
@@ -291,7 +301,12 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName schemaTableName,
             if (e.getStatusCode() == 404) {
                 // to indicate that we can't read from topic because there is no schema
                 return null;
+            } else if (e.getStatusCode() == 401) {
+                throw new PrestoException(QUERY_REJECTED,
+                        String.format("Failed to get schema information for topic %s/%s: Unauthorized",
```

Comment:
nit: schema has a different meaning in pulsar. so I would suggest being specific here: "Failed to get pulsar schema information"